SU-SWS/stanford_profile
# D8CORE-5763 Updated default content

# READY FOR REVIEW

# Summary
- Replace/add/remove some default content.

# Need Review By (Date)
- :shrug: 

# Urgency
- low

# Steps to Test
1. checkout this branch
2. do a fresh install
3. verify the default content still exists and there are some new taxonomy terms in shared tags

# See Also
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
